src/redux/configureStores.js

```diff
@@ -1,11 +1,11 @@
 import { configureStore, combineReducers } from '@reduxjs/toolkit';
-import books from './books/books';
-import categories from './categories/categories';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
 
```

Comment:
- [ ] (Optional) It would be nice to import `applyMiddleware` . It will enable you to use some handy Redux middleware, like a redux `logger`. The redux `logger` helps you in the development process, gives you an inner working of redux and also provides you with the previous state and current state when you trigger actions. All these can be seen in the `console` of the browser. 👍